docs/team/sohenze.md

```diff
@@ -3,22 +3,106 @@ layout: page
+    
+    * Feature: **Sort Order Command**
+    * Function: Allows the user to sort the displayed order list in the application based on the orders' time
+    of creation.
+    * Purpose: Supplementary feature of an application that manages orders which provides users further ease of use.
+    * Notable Implications:
+      * The application previously displayed the order list to the user in the form of a FilteredList under JavaFX
+      collections.
+      * This is because the application supports filtering the displayed order list according to keywords provided
+      by the user in another feature implemented by a teammate. FilteredList does not support any modifications to it.
+      * Wrapped the existing FilteredList within a SortedList under JavaFX collections and displayed the SortedList
+      to the user instead. This supported both features and allowed the user to sort filtered orders as well.
+    * Relevant pull request(s): [#108](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/108)
 
-    - **Contributions to team-based tasks**: to be added soon
+* **Contributions to the UG**: to be added soon
+  * Features Section:
+    * Deleting an order
+    * Deleting an item
+    * Editing an item
+    * Sorting orders
+  * Relevant pull request(s): [#101](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/101),
+  [#108](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/108),
+  [#203](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/203)
 
-    - **Review/mentoring contributions:**: to be added soon
+* **Contributions to the DG**:
+  * Design Section:
+    * Architecture
+    * UI
+    * Logic
+    * Model
+    * Storage
+    * Implementation Section:
+      * Delete Item Feature
+      * Edit Item Feature
+      * Sort Orders Feature
+  * Relevant pull request(s): [#114](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/114),
+  [#137](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/137),
+  [#203](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/203)
+  
+* **Contributions to team-based tasks**:
+  * Facilitated team discussions on design implementations
+  * Provided insights and proposed solutions to problems during team discussions
 
-    - **Contributions beyond the project team:**: to be added soon
-
-    - **Contributions to the Developer Guide (Extracts)**: to be added soon
-
-    - **Contributions to the User Guide (Extracts)**: to be added soon
+* **Review/mentoring contributions:**:
+  * Provided alternative opinions to teammates' contributions
+    * Relevant pull request(s): [#105](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/105)
+  * Provided explanations to teammates regarding personal contributions
+    * Relevant pull request(s): [#123](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/123),
+    [#128](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/128)
+  * Assisted teammates in fixing bugs introduced after merging different pull requests
```

Comment:
I think for this, can add the PR where you assisted me with editing the tests for `edito`!

Reply:
Yep added it in, thank you!